### PR TITLE
Display board

### PR DESCRIPTION
**Pull request by whaley-ethan**
This is to fulfill the story of Display Empty Board:
As a user When I start the program I see a representation of an empty board

### CODE REVIEW

File: src/main/java/tic/tac/toe/App.java [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-    }
+  private Game game;
+  App() {
+    game = new Game();
+  }
 
-    public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
-    }
+  public String displayGame() {
```
Comment (jchung722) [inline src/main/java/tic/tac/toe/App.java]: is this being used?

File: src/main/java/tic/tac/toe/App.java [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+    String board = game.displayGame();
+    System.out.println(board);
+    return board;
+  }
+
+  public String getGreeting() {
```
Comment (jchung722) [inline src/main/java/tic/tac/toe/App.java]: is this being used?

File: src/main/java/tic/tac/toe/App.java [hunk 3]
```diff
@@ -1,14 +1,24 @@
...
+    return new Game().getGreeting();
+  }
+
+  public static void main(String[] args) {
+    Game game = new Game();
```
Comment (jchung722) [inline src/main/java/tic/tac/toe/App.java]: `game` is already set above

File: src/test/java/tic/tac/toe/GameTest.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  @Test
+  public void testGameHasAGreeting() {
+    Game game = new Game();
+    assertNotNull("game should have a greeting", game.getGreeting());
+  }
+  
+  @Test
+  public void testNewGameBoardIsEmpty() {
+    Game game = new Game();
+    assertEquals("game should have an empty board", game.displayGame(), "   |   |   \n---|---|---\n   |   |   \n---|---|---\n   |   |   ");
```
Comment (jchung722) [inline src/test/java/tic/tac/toe/GameTest.java]: I think it's okay to have test description just once

File: src/test/java/tic/tac/toe/GameTest.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+public class GameTest {
+  @Test
+  public void testGameHasAGreeting() {
+    Game game = new Game();
+    assertNotNull("game should have a greeting", game.getGreeting());
+  }
+  
+  @Test
+  public void testNewGameBoardIsEmpty() {
```
Comment (jchung722) [inline src/test/java/tic/tac/toe/GameTest.java]: It's good practice to have test names describing the behavior of the code vs. any structure of the content. i.e. `NewGameDisplaysEmptyBoard`

File: src/test/java/tic/tac/toe/GameTest.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
+package tic.tac.toe;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class GameTest {
+  @Test
+  public void testGameHasAGreeting() {
+    Game game = new Game();
+    assertNotNull("game should have a greeting", game.getGreeting());
```
Comment (jchung722) [inline src/test/java/tic/tac/toe/GameTest.java:10]: test should be closer to its description

**Comment by jchung722** [comment]
Seems like a good opportunity to play around with interactive rebasing & squashing commits :) https://medium.com/@lucaspenzeymoog/intro-to-interactive-rebasing-in-git-and-customizing-vim-preferences-b6e2f0309e31

File: src/main/java/tic/tac/toe/App.java (hunk 1 above)
Comment (whaley-ethan) [reply src/main/java/tic/tac/toe/App.java]: No. Removed.

**Review by jchung722 — APPROVED** [review]

**Pull request closed without merging**